cbfeeds/feed.py

```diff
@@ -87,46 +95,67 @@ def validate(self, pedantic=False, serialized_data=None):
 class CbFeedInfo(object):
     def __init__(self, **kwargs):
         # these fields are required in every feed descriptor
-        self.required = ["name", "display_name", "version",
```

Comment:
Is the version attribute no longer required in feeds? I would keep it in there anyway for compatibility with previous versions of Cb- I seem to remember feeds not being accepted without the version attribute (but I may be mistaken; I haven't tried it in a while)